test/lists/list-item-spec.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import ListItem from 'lists/list-item';
+import injectTheme from '../fixtures/inject-theme';
+import TestUtils from 'react-addons-test-utils';
+
+describe('ListItem', () => {
+  let ThemedListItem;
+
+  beforeEach(() => {
+    ThemedListItem = injectTheme(ListItem);
+  });
+
+  it('should display list-item in it\'s default state', () => {
+    let render = TestUtils.renderIntoDocument(
+      <ThemedListItem
+      />
+    );
+    let listItem = TestUtils.findRenderedComponentWithType(render, ListItem);
+    expect(listItem.props.disabled).to.be.false;
```

Comment:
I feel like it's only testing `getDefaultProps`. I think that you can remove this test.